android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -25,6 +26,8 @@ class PlayerModule(context: ReactApplicationContext) : BitmovinBaseModule(contex
      */
     private val players: Registry<Player> = mutableMapOf()
 
+    var mediaSessionPlaybackManager: MediaSessionPlaybackManager? = null
```

Comment:
As discussed offline, let's init the `mediaSessionPlaybackManager` once here, instead of every time there is a player created with media controls support enabled.

Reply:
Address in cf7c34f41d4b752a1d2209df1029babd30b823e9 and finally in f0b06d7f338a79ed86b7567b248400a773bad994